website/docs/sdks/proxy-javascript.md

```diff
@@ -3,43 +3,49 @@ id: proxy-javascript
 title: JavaScript Proxy SDK
 ---
 
-In this guide we explain how to use feature toggles in a Single Page App via [The Unleash Proxy](/sdks/unleash-proxy). You can also checkout the source code for the [JavaScript Proxy SDK](https://github.com/unleash/unleash-proxy-client-js).
+In this guide we explain how to use feature toggles in a Single Page App via [Unleash Proxy](/sdks/unleash-proxy) or directly to Unleash (since _v4.16_). You can also checkout the source code for the [JavaScript Proxy SDK](https://github.com/unleash/unleash-proxy-client-js).
 
 ## Introduction {#introduction}
 
-For single-page apps we have a tiny proxy-client in JavaScript, without any external dependencies, except from browser APIs. This client will store toggles relevant for the current user in local-storage and synchronize with the Unleash Proxy in the background. This means we can bootstrap the toggles for a specific use the next time the user visits the web-page.
+For single-page apps we have a tiny proxy-client in JavaScript, without any external dependencies, except from browser APIs. This client will store toggles relevant for the current user in local-storage and synchronize with the Unleash in the background. This means we can bootstrap the toggles for a specific use the next time the user visits the web-page.
```

Comment:
```suggestion
The JavaScript proxy client is a tiny Unleash client written in JavaScript without any external dependencies (except from browser APIs). This client stores toggles relevant for the current user in `localStorage` and synchronizes with Unleash (the proxy _or_ the direct access API) in the background. Because toggles are stored in the user's browser, the client  can use them to bootstrap itself the next time the user visits the same web page.
```